### DISCUSSION TITLE
Convert Integer in Int16, Int32, UInt16 or UInt32

### DISCUSSION THREAD

**Discussion by plyzico** [FreeOpcUa/opcua-asyncio#726] [Q&A] [answered]
Hello,

I think this is an easy one, but I can't set it up. I'm trying to declerate one of the special Int variable (see headline). But I don't know how. 
For example I have i=30 
How can I convert it e.g. into Int16 or UInt16?

Thank you!

**Comment by AndreasHeine** [comment, marked as answer]
you can declare the Variant with typedefinition you need! opc ua ist strict typed.

**Reply by AndreasHeine** [reply]
write_value(ua.Variant(30, ua.VariantType.UInt16))

**Reply by plyzico** [reply]
Ok now I got it! 
I just used the casting:

i =30
value = Int16(i)

Tank you very much!
